azkaban-web-server/src/main/java/azkaban/flowtrigger/quartz/FlowTriggerScheduler.java

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright 2017 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.flowtrigger.quartz;
+
+import static java.util.Objects.requireNonNull;
+
+import azkaban.flow.Flow;
+import azkaban.flow.FlowUtils;
+import azkaban.project.FlowLoaderUtils;
+import azkaban.project.FlowTrigger;
+import azkaban.project.Project;
+import azkaban.project.ProjectLoader;
+import azkaban.project.ProjectManagerException;
+import azkaban.scheduler.QuartzJobDescription;
+import azkaban.scheduler.QuartzScheduler;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.io.Files;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import org.quartz.JobDataMap;
+import org.quartz.JobDetail;
+import org.quartz.JobKey;
+import org.quartz.Scheduler;
+import org.quartz.SchedulerException;
+import org.quartz.Trigger;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Singleton
+public class FlowTriggerScheduler {
+
+  private static final Logger logger = LoggerFactory.getLogger(FlowTriggerScheduler.class);
+  private final ProjectLoader projectLoader;
+  private final QuartzScheduler scheduler;
+
+  @Inject
+  public FlowTriggerScheduler(final ProjectLoader projectLoader, final QuartzScheduler scheduler) {
+    this.projectLoader = requireNonNull(projectLoader);
+    this.scheduler = requireNonNull(scheduler);
+  }
+
+  /**
+   * Schedule flows containing flow triggers
+   */
+  public void scheduleAll(final Project project, final String submitUser)
```

Comment:
why this method is called `scheduleAll` ?

Reply:
schedule all flows with flow trigger